build/ios.js

```diff
@@ -80,7 +54,6 @@ IOS.prototype.package = function (packager, next) {
 	const DEST_IOS = path.join(packager.zipSDKDir, 'iphone');
 
 	async.parallel([
-		this.fetchLibTiCore.bind(this),
```

Comment:
Can we also remove libti_ios_debugger.a? libti_ios_profiler.a?

Those were to be used against TiCore and do not work with JSCore.

I've been told several times we can kill off our profiler because it's not useful and no one uses/knows about it. The debugger has been replace by using the official debugger APIs built into JSCore.